GetDKAN/dkan
# Update for create-react-app config

* Brings the new create-react-app frontend into DKAN as a default
* Passes some config to the new (4.x) version of dkan-tools through the composer.json file
* Updates the tests and CI to use the new DKAN tools including the new proxy service for development domains

## QA Steps

* Install version 4.0.0 of DKAN Tools
* `dktl init --dkan=dev-add-cra-config`
* `dktl demo`
* Confirm you have a working site w/frontend
